tests/job.go

```diff
@@ -121,8 +121,8 @@ func NewHelloWorldJob(host string, port string, checkConnectivityCmdPrefixes ...
 // However, netcat does not work well with UDP and closes before the answer arrives, we make netcat wait until
 // the defined timeout is expired to prevent this from happening.
 func NewHelloWorldJobUDP(host, port string, checkConnectivityCmdPrefixes ...string) *batchv1.Job {
-	timeout := 3
-	check := fmt.Sprintf(`set -ex
+	timeout := 5
+	check := fmt.Sprintf(`set -x
```

Comment:
You've added these in the previous commit. Please squash these changes one into it.

Reply:
Done